docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -75,17 +75,23 @@ NOTE: Any changes to the thread pools require a restart of the {page-component-t
 This pool determines the number of requisitions that can be imported at one time.
 Increase this number if you have a large number of requisitions.
 
+Import threads will check the requisition against the database to determine what add/change/delete operations are necessary.
+They will also attempt to pull SNMP system information from the node into the database.
```

Comment:
... pull Node attributes from the SNMP system table. (I know there was a reason we did this on the import thread but I can't remember why for the life of me. I know it had something to do with the system object ID but I just can't recall why it had to happen on the import thread. Something about how it gets scheduled for rescans I'm sure.

Reply:
Maybe so that data is available before it gets to checking policies?